src/avt/MIR/Tet/Tetrahedralizer.C

```diff
@@ -7,12 +7,10 @@
 #include <vtkCell.h>
 #include <VisItException.h>
 
+#include <algorithm>
 #include <vector>
 using std::vector;
 
-#define MIN(A,B)  (((A) < (B)) ? (A) : (B))
-#define SWAP(A,B) { int swtmp; swtmp=A; A=B; B=swtmp; }
```

Comment:
I guess `SWAP` was defined but not used here?

Reply:
correct.